<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessDlg</class>
 <widget class="QWidget" name="ImageProcessDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageProcess</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>0</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>device control</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="open_device_btn">
       <property name="text">
        <string>open device</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="close_device_btn">
       <property name="text">
        <string>close device</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="acquisition_start_btn">
       <property name="text">
        <string>acquisition start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="acquisition_stop_btn">
       <property name="text">
        <string>acquisition stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>select device</string>
   </property>
   <widget class="QComboBox" name="device_list_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>trigger</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>trigger Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="trigger_mode_box"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>trigger software</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="trigger_software_btn">
         <property name="text">
          <string>send command</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>trigger source</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="trigger_source_box"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>280</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>color correction</string>
   </property>
   <widget class="QCheckBox" name="color_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>enable</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>sharpness</string>
   </property>
   <widget class="QSlider" name="sharpness_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QCheckBox" name="sharpness_box">
     <property name="text">
      <string>enable</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sharpness_edit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>400</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Gamma</string>
   </property>
   <widget class="QSlider" name="gamma_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="gamma_edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="objContrastBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>contrast</string>
   </property>
   <widget class="QSlider" name="contrast_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="contrast_edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>540</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>brightness</string>
   </property>
   <widget class="QSlider" name="lightness_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lightness_edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-150</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>150</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>610</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>basic parameter settings</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>71</width>
      <height>57</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="exposure_time_edit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="gain_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>32</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>exposure</string>
    </property>
   </widget>
   <widget class="QLabel" name="exposure_range">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>32</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>gain</string>
    </property>
   </widget>
   <widget class="QLabel" name="gain_range">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="show_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>280</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>save image</string>
   </property>
   <widget class="QPushButton" name="save_img_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
